static/app/views/replays/replays.tsx

```diff
@@ -254,4 +251,4 @@ const SortLink = styled(Link)`
   }
 `;
 
-export default withRouter(withPageFilters(withOrganization(Replays)));
+export default withPageFilters(Replays);
```

Comment:
Hmm do we have no `usePageFilters` hook?

Reply:
Intellisense didn't bring up any hooks related to it that. While I'm resolving the other feedback I'll take another look.

Reply:
Closest thing I could find was useProfileFilters, which returns PageFilters but also requires it as a param for the hook to work.

Reply:
cc @davidenwang can we add a `usePageFilters` hook so we can get rid of this HoC?

Reply:
Typically we've just used `useLegacyStore(PageFiltersStore)` as the hook